scripts/screeps_tencent_batch_rl_runner.py

```
@@ -4545,15 +4559,15 @@ def validate_static_inputs(args: argparse.Namespace, run_id: str) -> None:
     scenario_id = scenario_id_from_args(args)
     if scenario_id not in SCENARIO_IDS:
         raise BatchRunError(f"scenario id must be one of: {', '.join(SCENARIO_IDS)}")
-    if getattr(args, "require_multi_tier_scenario", False) and scenario_id != MULTI_TIER_SCENARIO_ID:
+    if getattr(args, "require_multi_tier_scenario", False) and scenario_id not in MULTI_TIER_SCENARIO_IDS:
         raise BatchRunError("multi-tier policy comparisons require the multi-tier territory/combat scenario id")
-    if getattr(args, "training_approach", None) == "policy_gradient" and scenario_id != MULTI_TIER_SCENARIO_ID:
+    if getattr(args, "training_approach", None) == "policy_gradient" and scenario_id not in MULTI_TIER_SCENARIO_IDS:
         raise BatchRunError(
             f"policy_gradient Tencent proof requires --scenario-id {MULTI_TIER_SCENARIO_ID} "
             "--require-multi-tier-scenario"
         )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)